esx_service/utils/error_code.py

```diff
@@ -92,7 +93,8 @@ class ErrorCode:
 
     ErrorCode.INTERNAL_ERROR : "Internal Error({0})",
     ErrorCode.INVALID_ARGUMENT : "Invalid Argument({0})",
-    ErrorCode.VOLUME_NAME_INVALID : "Volume name {0} is invalid, only {1} is allowed"
+    ErrorCode.VOLUME_NAME_INVALID : "Volume name {0} is invalid, only {1} is allowed",
+    ErrorCode.FEATURE_NOT_SUPPORT : "This feature is not supported for vmgroup {0}."
```

Comment:
No need for positional ({0}). `"This feature is not supported for vmgroup {}."` would be fine